integrations/gcp/CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- towncrier release notes start -->
 
+## 0.1.46 (2024-08-26)
+
+
+### Improvements
+
+- Introduced AsyncGeneratorRetry, a custom retry mechanism designed to handle entire resource retrieval retries instead of individual page retries. This change addresses the recurring DEADLINE_EXCEEDED errors that occurred with GCP's AsyncRetry during page-based retries, significantly improving reliability in GCP integrations.
+- Implemented a sophisticated rate limiting system based on quota levels, allowing for precise control over request limits per project, per service, and per quota.
+- Added a `SEARCH_ALL_RESOURCES_PER_MINUTE_QUOTA` environment variable for controlling the rate limit quota should the integration fail to query quota from GCP as stated in the last point. Default is 400
```

Comment:
in the code you use the parameter from config while here you are referring to environment variable

Reply:
In the code I'm actually fetching from `integration_config`.  the env ?🤔